packages/lodestar/src/chain/blocks/process.ts

```diff
@@ -75,7 +78,12 @@ export async function getPreState(
     pool.addPendingBlock(job);
     return null;
   }
-  return await db.stateCache.get(parentBlock.stateRoot as Uint8Array);
+  const context = await db.stateCache.get(parentBlock.stateRoot as Uint8Array);
+  if(!context.epochCtx) {
```

Comment:
I think we always have epochCtx in stateCache except for genesis state.
Even when we restart node, we also go thru this process and we have epochCtx.

We can either handle specifically for genesis state, or if we want to have fallback like this then probably it's better to put in `db.stateCache`